cmd/lotus-shed/indexes.go

```diff
@@ -92,8 +103,12 @@ var backfillEventsCmd = &cli.Command{
 			return err
 		}
 
+		log.Infof(
```

Comment:
No, and we'd end up with a race anyway, unless it was something like "give me a lock". It would be nice to do this all via RPC so we don't have this problem but I'm not sure an entirely new API just for this makes any sense.